### PR TITLE
New dockerfile including drone dependencies

### PR DESCRIPTION
**Pull request by pariaspe**
#670

### CODE REVIEW

**Comment by pariaspe** [comment]
Hi @SakshayMahna, check this new dockerfile. Thanks

File: scripts/Dockerfile [hunk 1]
```diff
@@ -11,12 +11,197 @@ ENV DEBIAN_FRONTEND noninteractive
...
 # and set the environment variables
-RUN mkdir -p /opt/jderobot && cd /opt/jderobot && git clone https://github.com/JdeRobot/CustomRobots.git && cd CustomRobots && git checkout melodic-devel && git pull origin melodic-devel
-ENV GAZEBO_MODEL_PATH=/opt/jderobot/CustomRobots/f1:/opt/jderobot/CustomRobots/roomba_robot
+RUN mkdir -p /opt/jderobot && cd /opt/jderobot && git clone https://github.com/JdeRobot/CustomRobots.git \
+    && cd CustomRobots && git checkout melodic-devel && git pull origin melodic-devel
+
+## Setting env vars
+ENV GAZEBO_RESOURCE_PATH "$GAZEBO_RESOURCE_PATH:/RoboticsAcademy/exercises/follow_line/web-template/launch"
+ENV GAZEBO_PLUGIN_PATH "$GAZEBO_PLUGIN_PATH:/opt/jderobot/CustomRobots/f1:/opt/jderobot/CustomRobots/roomba_robot:/Firmware/build/px4_sitl_default/build_gazebo"
+ENV GAZEBO_MODEL_PATH "$GAZEBO_MODEL_PATH:/Firmware/Tools/sitl_gazebo/models"
```
Comment (SakshayMahna) [inline scripts/Dockerfile:203]: Slight correction

`GAZEBO_MODEL_PATH` should contain `/opt/jderobot/CustomRobots/f1:/opt/jderobot/CustomRobots/roomba_robot`. By mistake, these directories are contained in `GAZEBO_PLUGIN_PATH`

**Review by SakshayMahna — CHANGES_REQUESTED** [review]
Just a small correction with `GAZEBO_MODEL_PATH` environment variable. Rest everything is good to go!

File: scripts/Dockerfile (hunk 1 above)
Comment (pariaspe) [reply scripts/Dockerfile]: Done! Thanks for noticing!

**Review by SakshayMahna — APPROVED** [review]

**Pull request merged**